drivers/scale_service.go

```diff
@@ -35,19 +35,27 @@ func (s *ScaleServiceDriver) ValidatePayload(conf interface{}, apiClient client.
 		return http.StatusBadRequest, fmt.Errorf("ServiceId not provided")
 	}
 
+	if config.Min == nil {
+		return http.StatusBadRequest, fmt.Errorf("Minimum scale not provided")
+	}
+
+	if config.Max == nil {
+		return http.StatusBadRequest, fmt.Errorf("Maximum scale not provided")
+	}
+
 	service, err := apiClient.Service.ById(config.ServiceID)
 	if err != nil {
 		return http.StatusInternalServerError, errors.Wrap(err, "Error in getService")
 	}
 
-	if service.Kind != "service" {
-		return http.StatusBadRequest, fmt.Errorf("Can only create webhooks for Services. The supplied service is of type %v", service.Kind)
-	}
-
```

Comment:
The PR for checking service.Kind has been merged, and I realized the condition needs to be checked earlier, so included this change as well in this PR